src/main/java/org/locationtech/spatial4j/shape/jts/JtsGeometry.java

```diff
@@ -573,17 +573,16 @@ private static Geometry cutUnwrappedGeomInto360(Geometry geom) {
       return geom;
     assert geom.isValid() : "geom";
 
-    //TODO opt: support geom's that start at negative pages --
-    // ... will avoid need to previously shift in unwrapDateline(geom).
```

Comment:
Do you think you can address this second line of the comment you deleted?  if you look in` unwrapDateline(LineString)`, there will be a final shifting going on that I think the improvement you added here obsoletes.  Granted I wrote that 8 years ago so I'm a little unclear now looking back at it!

Reply:
I will take a look at it!